computer_vision/lenet_pytorch.py

```diff
@@ -54,20 +54,28 @@ def forward(self, image_array: numpy.ndarray) -> numpy.ndarray:
         return image_array
 
 
-def test_model() -> bool:
+def test_model(image_tensor: torch.tensor) -> bool:
     """
-    Test the model on a random input of size [64, 1, 32, 32]
+    Test the model on an input batch of 64 images
 
-    >>> test_model()
+    Args:
+        image_tensor (torch.tensor): Batch of Images for the model
+
+    >>> test_model(torch.randn(64, 1, 32, 32))
     True
 
     """
-    random_image = torch.randn(64, 1, 32, 32)
-    model = LeNet()
-    output = model(random_image)
+    try: 
+        model = LeNet()
+        output = model(image_tensor)
+    except:
```

Comment:
https://realpython.com/the-most-diabolical-python-antipattern

computer_vision/lenet_pytorch.py:71:5: B001 Do not use bare `except:`, it also catches unexpected events like memory errors, interrupts, system exit, and so on.

computer_vision/lenet_pytorch.py:71:5: E722 do not use bare 'except'

Reply:
Corrected